test/unit/alphabet/adaptation/uint_test.cpp

```diff
@@ -85,7 +91,7 @@ TYPED_TEST(uint_adaptation, assign_char)
     EXPECT_EQ((assign_char_to('C', l)), TypeParam{67});
 }
 
-TYPED_TEST(uint_adaptation, assign_char_strict)
+TYPED_TEST(uint_adaptation, assign_char_strictly_to)
```

Comment:
assign_rank and assign_char -> assign_rank_to and assign_char_to
:nail_care: since you changed this for the other test too